internals/docs/guides/peripheral-uart.md

```diff
@@ -0,0 +1,152 @@
+# UART
+
+[UART](https://en.wikipedia.org/wiki/Universal_asynchronous_receiver-transmitter) (Universal Asynchronous Receiver-Transmitter) is a programmable interface that allows you to communicate 1:1 with a UART peripheral.
+
+**Figure: UART interface diagram**
+
+![UART interface diagram](media/peri_api_uart_diagram.png)
+
+UART is an interface for exchanging data with peripherals. It is an asynchronous method that transmits data without a clock line, and consists of only 2 data lines: transmit (Tx) and receive (Rx). UART performs 1:1 communication.
+
+**Figure: UART data frame**
+
+![UART data frame](media/peri_api_uart_data.png)
+
+The UART data frame consists of start (1 bit), data (5\~8 bit), parity (1 bit) and stop (1 bit):
+
+-   Start bit
+
+    Indicates the start of the communication before sending data and holds it for a bit time length.
+
+-   Data
+
+    Transmits 5 to 8 bits of data.
+
+-   Parity bit
+
+    Generates and transmits a parity value for error verification, and determines a receiving side error. The following options are available: `None`, `Even`, and `Odd` parity. Selecting `None` removes this bit.
+
+-   Stop bit
+
+    Indicates the termination of the communication and holds 1 or 2 bits.
+
+-   Baud rate
+
+    Asynchronous transmission/receiving speeds must be matched to the peripheral. For this purpose, the number of signals transmitted per second can be synchronized with a peripheral. It is called the Baud.
+
+If a device supports a 5-wire UART port, hardware flow control can be used to increase the reliability of the data transmission. Software flow control can also be used to increase reliability.
+
+## Opening and Closing a Handle
+
+To open and close a handle:
+
+1.  To open a UART handle, create the `SerialPort()` object:
+
+    ```csharp
+    int port = 0;
+    SerialPort uart = new SerialPort(port);
+    ```
+
+    The `port` parameter required for this function must be set according to the following tables.
+
+    **Table: Raspberry Pi 3 and Raspberry Pi 4**
+
+    | Pin name | | Port (parameter 1) |
+    | -------- | ---------| ---- |
+    | UART0_RX | UART0_TX | 0    |
```

Comment:
Can you re-check whether the port is correct?
In my memory, two UART port were supported in RPI4.

Reply:
thanks I will update docs.